src/main/java/com/odysseusinc/arachne/datanode/dto/atlas/Pathway.java

```diff
@@ -0,0 +1,155 @@
+package com.odysseusinc.arachne.datanode.dto.atlas;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import com.odysseusinc.arachne.datanode.dto.serialize.MultiFormatDateDeserializer;
+import java.util.Date;
+import java.util.List;
+
+public class Pathway extends BaseAtlasEntity {
```

Comment:
Why don't you use classes from [StandardizedAnalysisAPI](https://github.com/OHDSI/StandardizedAnalysisAPI/tree/master/standardized-analysis-specs/src/main/java/org/ohdsi/analysis/pathway)

Reply:
Well, but StandardizedAnalysisAPI provides only intefaces rather than implementations.
I not feel any profit of StandardizedAPI interfaces implementation for internal DTO classes.